server/src/main/java/org/elasticsearch/index/codec/vectors/diskbbq/IVFVectorsReader.java

```diff
@@ -339,6 +339,7 @@ public final void search(String field, float[] target, KnnCollector knnCollector
             }
         }
         if (acceptDocsBits != null) {
+            // TODO Adjust the value here when using centroid filtering
             float unfilteredRatioVisited = (float) expectedDocs / numVectors;
```

Comment:
While I expect this to impact things, I am not sure what we can do. I would presume that our recall curve is WAY better now with eager filtering, and users can recover the (0.1%?) change in recall by asking for more exploration.